packages/ui/components/card/FormCard.tsx

```diff
@@ -61,16 +61,14 @@ export default function FormCard({
               </Badge>
             )}
           </div>
-          {deleteField?.check() ? (
```

Comment:
Why was it removed? It results in showing delete button where it doesn't work. 

![image](https://github.com/calcom/cal.com/assets/1780212/60d237db-b7c1-4cc8-b405-6dd9402fbfd6)